service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -185,7 +188,8 @@ public void deletingAnExistingPolicy() throws Exception {
   public RequestResponsePact getPaoWithAnExistingPolicy(PactDslWithProvider builder) {
     var policyResponseShape =
         new PactDslJsonBody()
-            .valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId)
+            .valueFromProviderState(
+                "objectId", existingPolicyProviderStateValue, existingPolicyId.toString())
```

Comment:
@blakery this did the trick!  Thanks for figuring out that the `toString()` was essential.